gulpfile.js

```diff
@@ -177,7 +177,7 @@ gulp.task('fonts', () => {
  * Creates a sourcemap.
  ******************************************************************************/
 
-gulp.task('js', ['js:hint', 'js:cs', 'browserify-tests'], () => {
+gulp.task('js', ['js:hint', 'js:cs'], () => {
```

Comment:
see https://github.com/elifesciences/pattern-library/pull/852 may also not be strictly necessary here